app/services/utm_service.rb

```diff
@@ -2,42 +2,43 @@
+    check_ect_and_mentor_info: "check-ect-and-mentor-info",
+    choose_materials: "choose-materials",
+    choose_provider: "choose-provider",
+    choose_route: "choose-route",
+    ect_validation_info_2109: "ect-validation-info-2109",
+    ect_validation_info_2709: "ect-validation-info-2709",
+    induction_coordinators_who_are_mentors_to_add_validation_information: "induction-coordinators-who-are-mentors-to-add-validation-information",
+    june_private_beta: "june-private-beta",
+    mentor_validation_info_2309: "mentor-validation-info-2309",
+    mentor_validation_info_2709: "mentor-validation-info-2709",
     new_admin: "new-admin",
+    new_induction_tutor: "new-induction-tutor",
     new_lead_provider: "new-lead-provider",
     nominate_tutor: "nominate-tutor",
     nominate_tutor_cip_only: "nominate-tutor-cip-only",
-    new_induction_tutor: "new-induction-tutor",
-    sign_in: "sign-in",
-    challenge_partnership: "challenge-partnership",
-    partnership_notification: "partnership-notification",
-    june_private_beta: "june-private-beta",
-    sign_in_reminder: "sign-in-reminder",
-    choose_route: "choose-route",
-    choose_provider: "choose-provider",
-    choose_materials: "choose-materials",
-    add_participants: "add-participants",
     participant_validation_beta: "participant-validation-beta",
     participant_validation_research: "participant-validation-research",
     participant_validation_sit_notification: "participant-validation-sit-notification",
-    check_ect_and_mentor_info: "check-ect-and-mentor-info",
-    induction_coordinators_who_are_mentors_to_add_validation_information: "induction-coordinators-who-are-mentors-to-add-validation-information",
-    sit_to_complete_steps: "sit-to-complete-steps",
-    year2020_nqt_invite_school: "year2020-nqt-invite-school",
-    year2020_nqt_invite_sit: "year2020-nqt-invite-sit",
-    ect_validation_info_2109: "ect-validation-info-2109",
-    ect_validation_info_2709: "ect-validation-info-2709",
-    mentor_validation_info_2309: "mentor-validation-info-2309",
-    mentor_validation_info_2709: "mentor-validation-info-2709",
-    sit_mentor_validation_info_2709: "sit-mentor-validation-info-2709",
-    year2020_nqt_invite_school_not_opted_out: "year2020-nqt-invite-school-not-opted-out",
-    year2020_nqt_invite_sit_validated: "year2020-nqt-invite-sit-validated",
-    year2020_nqt_invite_sit_no_participants: "year2020-nqt-invite-sit-no-participants",
     partnered_invite_sit_reminder: "partnered-invite-sit-reminder",
+    partnership_notification: "partnership-notification",
     remind_fip_sit_to_complete_steps: "remind-fip-sit-to-complete-steps",
-    add_participants_unpartnered_cip: "add-participants-unpartnered-cip",
+    sign_in: "sign-in",
+    sign_in_reminder: "sign-in-reminder",
+    sit_mentor_validation_info_2709: "sit-mentor-validation-info-2709",
+    sit_participant_email_bounced: "sit-participant-email-bounced",
```

Comment:
Do we not need to add this into the `SOURCE` as well as the `CAMPAIGN` hash?

Reply:
I don't think so, the GA campaigns now seem to be showing up as intended